Tasks/HelmInstallerV0/src/helminstaller.ts

```diff
@@ -68,15 +68,33 @@ function getHelmDownloadURL(version: string): string {
 }
 
 async function getStableHelmVersion(): Promise<string> {
-    var request = new WebRequest();
-    request.uri = helmLatestReleaseUrl;
-    request.method = "GET";
-
     try {
-        var response = await sendRequest(request);
-        return response.body["tag_name"];
+        const downloadPath = await toolLib.downloadTool(helmAllReleasesUrl);
```

Comment:
Why are you using downloadTool?

Reply:
This should be achievable using WebRequest too

Reply:
We were using downloadTool in helmUtility. I thought we should keep the code consistent everywhere. Also, the response now is quite large and that's why downloading it as a file seemed to be a better idea.

Reply:
ok